lib/Collection.js

```diff
@@ -78,7 +78,7 @@ module.exports = class Collection {
       return false;
     }
     const edvDoc = await this.getEdvDocument({id: existing.id});
-    return edvDoc.delete();
+    return edvDoc.delete({doc: existing});
```

Comment:
Does the above get the EDV document twice? If so, please refactor so it doesn't.

Reply:
@JSAssassin it looks like this actually does fetch the document twice. Can you confirm if that is true?

Reply:
@dlongley @aljones15 no, it doesn't fetch the document twice. edvDoc has the delete function on it while the other  `existing` doesn't.

Reply:
Also, I noticed the `Collection.js` file is being used in `bedrock-profile-manager` as well. Do you guys think it would be better to have a standalone library of it?

Reply:
@JSAssassin make an issue and link to the examples of multiple use. Anyways, I'm happy with your check on the double read so ping @dlongley and let's see if we can get this improvement into vc-issuer.

Reply:
>  no, it doesn't fetch the document twice. edvDoc has the delete function on it while the other existing doesn't.

That doesn't mean it doesn't fetch the document twice. I'm asking this: Does it hit the edv server twice here before it deletes the document? If so, that's a problem we need to fix. I believe this came up elsewhere and was similarly corrected. I'll see if I can figure out where. @mandyvenables may know.

Reply:
Ok, I've found similar code here:

https://github.com/digitalbazaar/bedrock-web-profile-manager/blob/d3e0e68214618591618cfb37eb9c1459eed9ee93/Collection.js#L74-L79

And saw it's the same. What's going on is that creating `existing` does require hitting the server, but `this.getEdvDocument` only creates an API wrapper -- which is then passed `existing` when calling `.delete`. So this only hits the server once. I'm going to file an issue for getting a better helper name than `getEdvDocument` because it's caused this confusion at least twice now.

I'll approve this PR.